OpenSim/Examples/ExampleHopperDevice/exampleHopperDevice.cpp

```diff
@@ -223,11 +223,11 @@ void run(bool showVisualizer, bool simulateOnce)
         hopper.finalizeFromProperties();
 
         // Show all Components in the model.
-        showSubcomponentInfo(hopper);
+        hopper.printSubcomponentInfo();
         // Show only the Joints in the model.
-        showSubcomponentInfo<Joint>(hopper);
+        hopper.printSubcomponentInfo<Joint>();
         // Show the outputs generated by the thigh body.
-        showAllOutputs(hopper.getComponent("/Dennis/thigh"), false);
+        hopper.getComponent("/Dennis/thigh").printOutputInfo(false);
```

Comment:
Since this example is used for teaching, perhaps include in the comment "Do not show…" to explain what the `false` argument means.